charts/posthog/values.yaml

```diff
@@ -21,8 +21,6 @@ sentryDSN:
 clickhouseOperator:
   # -- Whether to install clickhouse. If false, `clickhouse.host` must be set
   enabled: true
-  # -- Which namespace to install clickhouse operator to
-  namespace: "posthog"
```

Comment:
keep it just as empty, so we have the description. I believe the default function still works then

Reply:
perfect